pyvista/core/filters/data_set.py

```diff
@@ -3261,6 +3266,11 @@ def point_data_to_cell_data(self, pass_point_data=False, progress_bar=False):
         pass_point_data : bool, default: False
             If enabled, pass the input point data through to the output.
 
+        categorical : bool, default: False
+            Control whether the source point data is to be treated as
+            categorical. If the data is categorical, histograming will be used
```

Comment:
I don't know what "histograming" means here without more context.  In the vtk docs it says "If the data is categorical, then the resultant cell data will be determined by a "majority rules" vote (using a histogram of the point data scalar values), with ties going to the smaller point data value." which is much more understandable to me.  Can we says something like `If ``True``, then the cell data will be determined by "majority rules" vote, with ties going to smaller values.`?

Reply:
This is how they frame it in the VTK docs:

> [vtkPointDataToCellData](https://vtk.org/doc/nightly/html/classvtkPointDataToCellData.html) is a filter that transforms point data (i.e., data specified per point) into cell data (i.e., data specified per cell). By default, the method of transformation is based on averaging the data values of all the points defining a particular cell. Optionally (by enabling CategoricalData), histograming can be used to assign the cell data. For large datasets with several cell data arrays, the filter optionally supports selective processing to speed up processing. Optionally, the input point data can be passed through to the output as well.

Reply:
Just a suggestion from me.  Not a blocker.  I personally find the vtk definition in that section not very descriptive.

Reply:
> I personally find the vtk definition in that section not very descriptive.

This is also my personal assessment. When I think of histograms I think of *variable* bin sizes, e.g. there are 100 categories and use 5 bins with values [0, 20], [21-40], [41-60], etc. or maybe use 4 bins instead [0, 25], [26-50], etc. to group and visualize the data. But this is not what the categorical option is doing in this case, it just does a frequency count of the point scalars for each cell and keeps the value with the largest count. I found this comment in the vtk [source code]( https://github.com/Kitware/VTK/blob/3866322a8bea79a5dc6c99ffe312a8cb76326073/Filters/Core/vtkPointDataToCellData.cxx#L279-L280) more helpful than the docs for describing what "histograming" means:
> // Populate a histogram from the scalar values at each
> // point, and then select the bin with the most elements.

But, I still think it's best if we avoid using the term "histograming". I liked @MatthewFlamm's suggestion which is clear and succinct:

> If ``True``, then the cell data will be determined by "majority rules" vote, with ties going to smaller values.

Reply:
> When I think of histograms I think of _variable_ bin sizes, e.g. there are 100 categories and use 5 bins with values [0, 20], [21-40], [41-60], etc. or maybe use 4 bins instead [0, 25], [26-50], etc. to group and visualize the data. But this is not what the categorical option is doing in this case, it just does a frequency count of the point scalars for each cell and keeps the value with the largest count. 

As a follow-up, this method *does* seem to group values together, and not just do a simple frequency count ([source code](
https://github.com/Kitware/VTK/blob/3866322a8bea79a5dc6c99ffe312a8cb76326073/Filters/Core/vtkPointDataToCellData.cxx#L192-L193)):
``` c++
// If the adjacent bins are close enough to be merged...
if (std::abs((*it1).Value - (*it2).Value) < VTK_EPSILON)
```

with 
``` c++
#define VTK_EPSILON 1.e-6
```

But, this makes for a very narrow bin, and only really applies to continuous scalars. For actual categorical data, integers are typically used, in which case this is effectively a majority rules without any real "binning" or grouping in the histogram sense I described earlier.

Reply:
> As a follow-up, this method _does_ seem to group values together, and not just do a simple frequency count ([source code](https://github.com/Kitware/VTK/blob/3866322a8bea79a5dc6c99ffe312a8cb76326073/Filters/Core/vtkPointDataToCellData.cxx#L192-L193)):

Thanks for the in depth analysis.  I always thought the "histograming" comment was odd, but from the source code, it is the most accurate when including float-valued data.  I still like my original suggestion, as users should be avoid using this flag with float data.  It also makes the process more clear what happens for the more suitable integer-valued data.  But, I also don't feel strongly about this, since the original wording is technically correct and is what vtk uses.